worker/provisioner/lxc-broker_test.go

```diff
@@ -85,9 +87,14 @@ func (s *lxcBrokerSuite) SetUpTest(c *gc.C) {
 	c.Assert(err, jc.ErrorIsNil)
 }
 
+func (s *lxcBrokerSuite) TearDownTest(c *gc.C) {
```

Comment:
Adding this means that the `lxcSuite.TearDownTest` method is no longer called automatically. Please add a call to `s.lxcSuite.TearDownTest(c)` at the end.

BTW, there is a helper called PatchValue that can be used to temporarily change a value, resetting when the test case tears down. Assuming this suite embeds the right base suite, you can do `s.PatchValue(&version.Current.Arch, arch.AMD64)`